tests/integration/call_run_on_script_with_keras_fit.py

```diff
@@ -34,16 +34,24 @@
 #     stream_logs=True)
 
 # Automated MirroredStrategy: chief config with multiple GPUs
+# tfc.run(
```

Comment:
Why the disabled test?

Reply:
We can call tfc.run only once from a python script. Here i am showcasing the various distribution strategies using the same python script. So i have commented out all but one strategy.